### PR TITLE
Sass common fixes

### PR DESCRIPTION
**Pull request by paddyo**
Repairs to the sass-common library to allow it to be deployable

### CODE REVIEW

File: src/sass/_functions/units.scss [hunk 1]
```diff
@@ -105,24 +105,25 @@
 // // multiplier(2rem) -> 32px
 
 @function multiplier($val) {
-  
+
+  $REMsupport: false !default;
```
Comment (RazvanDH) [inline src/sass/_functions/units.scss]: do we need this? if $REMsupport is not present, nothing will work anyway

File: src/sass/mixins.scss [hunk 2]
```diff
@@ -10,6 +10,5 @@
 @import '_mixins/breakpoints';
 @import '_mixins/css-properties';
 @import '_mixins/fonts';
-// @import '_mixins/forms';
+@import '_mixins/forms';
```
Comment (RazvanDH) [inline src/sass/mixins.scss]: We don't need this for now. Can you please remove it? I left it commented just to remind myself to integrate the new stylings for the forms here. Thanks.

File: src/sass/_functions/units.scss (hunk 1 above)
Comment (paddyo) [inline src/sass/_functions/units.scss]: It throws a SASS error

File: src/sass/mixins.scss (hunk 2 above)
Comment (paddyo) [inline src/sass/mixins.scss]: There is a dependency on customCheckboxAndRadio in static-content, so it cannot be removed without a major version bump

**Pull request merged**